packages/@react-spectrum/breadcrumbs/stories/Breadcrumbs.stories.tsx

```diff
@@ -87,23 +88,23 @@ storiesOf('Breadcrumbs', module)
 function render(props = {}) {
   return (
     <Breadcrumbs {...props}>
-      <BreadcrumbItem>Folder 1</BreadcrumbItem>
-      <BreadcrumbItem>Folder 2</BreadcrumbItem>
-      <BreadcrumbItem>Folder 3</BreadcrumbItem>
+      <Item>Folder 1</Item>
+      <Item>Folder 2</Item>
+      <Item>Folder 3</Item>
     </Breadcrumbs>
   );
 }
 
 function renderPress(props = {}) {
   return (
     <Breadcrumbs {...props}>
-      <BreadcrumbItem onPress={action('press Folder 1')}>Folder 1</BreadcrumbItem>
-      <BreadcrumbItem onPress={action('press Folder 2')}>Folder 2</BreadcrumbItem>
-      <BreadcrumbItem onPress={action('press Folder 3')}>Folder 3</BreadcrumbItem>
-      <BreadcrumbItem onPress={action('press Folder 4')}>Folder 4</BreadcrumbItem>
-      <BreadcrumbItem onPress={action('press Folder 5')}>Folder 5</BreadcrumbItem>
-      <BreadcrumbItem onPress={action('press Folder 6')}>Folder 6</BreadcrumbItem>
-      <BreadcrumbItem onPress={action('press Folder 7')}>Folder 7</BreadcrumbItem>
+      <Item onPress={action('press Folder 1')}>Folder 1</Item>
+      <Item onPress={action('press Folder 2')}>Folder 2</Item>
+      <Item onPress={action('press Folder 3')}>Folder 3</Item>
+      <Item onPress={action('press Folder 4')}>Folder 4</Item>
+      <Item onPress={action('press Folder 5')}>Folder 5</Item>
+      <Item onPress={action('press Folder 6')}>Folder 6</Item>
+      <Item onPress={action('press Folder 7')}>Folder 7</Item>
```

Comment:
Would there ever be a time where an end user would have a generative item list rather than a static list like this? Might be nice to have a example for testing (also would be nice to check if passing onPress works in that case as well)

Reply:
Also feels like it might be nice if we could pass onPress to the `Breadcrumb` component instead and it would get applied to each item via useSelectableItem/Menu or something

Reply:
generative item list will be something like this:
```
    <Breadcrumbs items={flatItems}>
      {item => <Item>{item.name}</Item>}
    </Breadcrumbs>
```
but breadcrumbs are not using CollectionBuilder and cannot accept items as prop (for now)... so only static list is supported

Reply:
from original ticket https://jira.corp.adobe.com/browse/RSP-1134 API suggested that onPress is added to single breadcrumb item... adding onPress to the parent component seems useful, we can discuss about adding it

Reply:
Cool, I think this is fine for now. If others agree about having onPress on the parent component, maybe make a follow up ticket for that

Reply:
I don't think `onPress` is very useful on the parent because you'd have no idea what was pressed. If generating from a list, you can use `.map` and put an `onPress` on each item.